CONTRIBUTING.md

```diff
@@ -56,27 +66,19 @@ Go can be [installed](https://golang.org/doc/install#install) from the official
 
 ## Build Juju and its dependencies
 
-### Download Juju source
+The easiest way to get the Juju source code is to clone the GitHub repository:
 
     git clone https://github.com/juju/juju.git
 
-Juju uses Go modules and does not depend on GOPATH, therefore you can check juju out anywhere.
-
-### Change to the Juju source code directory
-
-    cd juju
-
-### Install runtime dependencies
-
-    make install-dependencies
-
-### Compile
-
-    make build
-
-### Install
-
-    make install
+To build/install from source, `cd` into the root directory of the cloned repo,
+and use `make`.
+- `make go-build` will build the Juju binaries and put them in a
+  `_build` subdirectory.
+- `make go-install` will build the Juju binaries and install them globally
```

Comment:
It's not really globally. What about changing "globally on your machine" to "in your `$GOBIN` directory (falling back to `$GOPATH/bin` or `~/go/bin`)".

Reply:
Good call